stdlib/public/Concurrency/Task.swift

```diff
@@ -76,42 +90,34 @@ public struct Task<Success, Failure: Error>: Sendable {
 
 @available(SwiftStdlib 5.5, *)
 extension Task {
-  /// Wait for the task to complete, returning (or throwing) its result.
+  /// Wait for the task to complete, returning its result or throwing an error.
   ///
-  /// ### Priority
-  /// If the task has not completed yet, its priority will be elevated to the
-  /// priority of the current task. Note that this may not be as effective as
-  /// creating the task with the "right" priority to in the first place.
+  /// If the task hasn't completed,
```

Comment:
```suggestion
  /// If the task hasn't completed yet,
```

The yet somehow helps me understand what we mean here, wdyt?

Reply:
Sure, that's more specific. Added in 946c72cbc73.